### ISSUE TITLE
Host can change currency

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#2720]
[Flarum](https://opencollective.com/flarum) has recently moved to their own fiscal host from OSC. Because they are in Europe, their currency is showing in Euros, but they actually want to stay with US dollars. How can they change the currency? I can't find this setting in the Host or Collective settings.

**Comment by luceos** [comment]
A few unwanted side effects:

- Backers were stored with USD, see [this](https://github.com/opencollective/opencollective/issues/981#issuecomment-377568011).
- Backers on a tier no longer are on a tier. Seeing active backers is now only possible using the API.

I am seriously hoping we didn't just lose all recurring donations, because that would be a huge hit to our financial stability and thus our sustainability. We are moving to get one core developer to work for us part time as a freelancer, we need all these funds for running costs and his salary!

To be honest, I actually wrote @alanna (who could have never known this) about my fears that something bad could happen and demanded clarification about the exact procedure. Even though the procedure was simple, it struck me as odd that after changing fiscal host I could only "add funds" in Euro's and no longer as dollars. If I had known in advance it would have made this a lot easier and maybe I would have decided against switching to our own fiscal host. This exact thing happened with [Tenancy back then](https://github.com/opencollective/opencollective/issues/981), so no one actually tested this and documented the oddities of doing the procedure. For a platform working with money I'm just hoping that at some point you will see the wisdom of making things like this less of a surprise to anyone.

tldr: Can anyone tell me:

- Can we (easily) switch back to USD if we wanted to? It really doesn't matter to Stripe, they will accept USD from OC and then pay us to our bank account in EUR.
- Did we just lose all recurring contributions?
- Can someone push the recurring backers that were on a tier, back into a tier? 

This is not meant as a rant, even though it feels a bit like it. I do love what OC is and does, I just feel disappointed (again).

**Comment by znarf** [comment]
> Can we (easily) switch back to USD if we wanted to? It really doesn't matter to Stripe, they will accept USD from OC and then pay us to our bank account in EUR.

Yes. I think keeping everything in USD is the sensible choice if the goal is to keep the current recurring contributions running. What behavior would the existing contributors expect?

> Did we just lose all recurring contributions?

No. According to the documentation, the subscriptions should be kept https://docs.opencollective.com/help/collectives/change-fiscal-host#what-happens-to-my-recurring-financial-contributions-followers-of-updates-and-transaction-history

Seems you created new Tiers when switching in euros (and deleting the old ones), the new Tiers are the one displayed and no one subscribed to them. The old tiers are deleted and not appearing, the attached subscriptions are still there.

> Can someone push the recurring backers that were on a tier, back into a tier?

Yes, that can be done.

**Comment by znarf** [comment]
Let's troubleshoot this issue over our support channels, happy to help on Slack or support@opencollective.com.

**Comment by znarf** [comment]
To come back to this ticket original purpose, this is not something that we want automated or done by the users.

- Collectives should always use the Host currency for accounting
- Collectives with past transactions should not be allowed to change their currency (they should be able to swap hosts but not with to an Host with a different currency)
- Hosts with past transactions should not be allowed to change their currency

This rules are dictated by the current limitations of our systems. In an ideal world we would be able to provide such flexibility.
